test/integration_test.go

```diff
@@ -266,7 +266,8 @@ func (ts *IntegrationTestSuite) TestBasic() {
 	ts.assertMetricCount("temporal_request", 1, "operation", "StartWorkflowExecution")
 	ts.assertMetricCountAtLeast("temporal_request", 1, "operation", "RespondWorkflowTaskCompleted")
 	ts.assertMetricCountAtLeast("temporal_workflow_task_queue_poll_succeed", 1)
-	ts.assertMetricCountAtLeast("temporal_long_request", 3, "operation", "PollActivityTaskQueue")
+	// We cannot check PollActivityTaskQueue metric because eager activities
+	// affect poll count
```

Comment:
I think that's gotta get fixed - it makes the metric not representative of what it says on the tin

Reply:
The metric is representative though. We have a metric for how many activity polls we call on the server and we asserted on that count in this test. We no longer do that assertion because we don't call poll as many times anymore w/ eager activities.